### PR TITLE
Lu sure/movie seat booking

### PR DESCRIPTION
**Pull request by Lu-sure**
**Movie seat booking task**

[Demo](https://lu-sure.github.io/Movie_seat_booking/)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/Lu-sure/movie-seat-booking/index.html [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@400;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="booking-wrapper">
+        <section class="film-info-wrapper">
+            <h2 class="film-name">"The Silence of the Lambs"</h2>
+            <div class="film-info-container">
+                <time datetime="2021-02-14" class="film-info">Feb 14, 2021</time>
+                <span class="slash">|</span>
```
Comment (A-Ostrovnyy) [inline submissions/Lu-sure/movie-seat-booking/index.html]: You can use pseudoelement for it)

File: submissions/Lu-sure/movie-seat-booking/index.html [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+                <time datetime="PT1H58M" class="film-info">1h 58min</time>
+                <span class="slash">|</span>
+                <span class="film-info">2D</span>
+                <span class="slash">|</span>
+                <span class="film-info">Crime, Drama, Thriller</span>
+            </div>
+
+            <img
+                src="img/poster.jpg"
+                alt="the pale woman's face with brown eyes and brown-yellow butterfly on lips"
```
Comment (A-Ostrovnyy) [inline submissions/Lu-sure/movie-seat-booking/index.html]: 🔥

File: submissions/Lu-sure/movie-seat-booking/index.html [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Movie Seat Booking</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@400;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="booking-wrapper">
+        <section class="film-info-wrapper">
+            <h2 class="film-name">"The Silence of the Lambs"</h2>
```
Comment (A-Ostrovnyy) [inline submissions/Lu-sure/movie-seat-booking/index.html]: Use `h1` tag for main heading of the page.

File: submissions/Lu-sure/movie-seat-booking/index.html [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+        <div class="credits">
+            Icons made by
+            <a href="https://www.freepik.com" title="Freepik">
+                Freepik
+            </a>
+            from
+            <a href="https://www.flaticon.com/" title="Flaticon">
+                www.flaticon.com
+            </a>
+        </div>
```
Comment (A-Ostrovnyy) [inline submissions/Lu-sure/movie-seat-booking/index.html]: 🔥

File: submissions/Lu-sure/movie-seat-booking/index.html [hunk 5]
```diff
@@ -0,0 +1,275 @@
...
+                        <span class="seat-icon"></span>
+                    </label>
+                    <label class="seat">
+                        <input type="checkbox" class="seat-checkbox visually-hidden" value="6-8">
+                        <span class="seat-icon"></span>
+                    </label>
+                    <span class="seat-row row-end">row 6</span>
+                </div>
+
+                <input class="confirm-but" type="submit" value="Book now">
```
Comment (A-Ostrovnyy) [inline submissions/Lu-sure/movie-seat-booking/index.html]: Add style s on `:hover`, `:active`, and `focus` states.

File: submissions/Lu-sure/movie-seat-booking/style.css [hunk 6]
```diff
@@ -0,0 +1,228 @@
...
+    color: white;
+    background-color: rgb(208, 227, 235);
+}
+
+.booking-wrapper {
+    display: flex;
+    flex-wrap: wrap;
+    width: 800px;
+    height: auto;
+    background-color: rgb(78, 104, 160);;
```
Comment (A-Ostrovnyy) [inline submissions/Lu-sure/movie-seat-booking/style.css]: Remove extra semicolon.)

File: submissions/Lu-sure/movie-seat-booking/style.css [hunk 7]
```diff
@@ -0,0 +1,228 @@
...
+    content: '';
+    display: inline-block;
+    margin-right: 10px;
+    width: 2em;
+    height: 2em;
+    background-size: cover;
+}
+
+.seat-available::before{
+
```
Comment (A-Ostrovnyy) [inline submissions/Lu-sure/movie-seat-booking/style.css]: Remove unused lines.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Nice work, iconic movie).
Remove files not for review(images, etc).
See my comment below!

**Comment by Lu-sure** [comment]
Changes are made)
I've installed prettier only now, so all the html shifted, my bad.
Thanks for your 🔥🔥

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**